forgerock-openbanking-uk-aspsp-rs/forgerock-openbanking-uk-aspsp-rs-mock-store/forgerock-openbanking-uk-aspsp-rs-mock-store-server/src/test/java/com/forgerock/openbanking/aspsp/rs/store/utils/FRCustomerInfoTestHelper.java

```diff
@@ -41,7 +42,7 @@ public static FRCustomerInfo aValidFRCustomerInfo() {
                 .id("3242-2343-23432-4323")
                 .userID("fredtitmus")
                 .address(FRCustomerInfoTestHelper.aValidFRCustomerInfoAddress())
-                .birthdate(new LocalDate())
+                .birthdate(new DateTime().minusYears(19))
                 .email("fred.titmus@forgerock.com")
```

Comment:
Changed birthdate from LocalDate to DateTime to avoid array representation when mapping to json to present the value on the UI.

Reply:
Surprised this doesn't make some unit tests fail as if the user data indicates that the user is under 18 we return an error.

Reply:
Did the IT tests pass with this change?

Reply:
Yes, all IT pass, I have introduced that change only in the FR object, we use the OBIE object to validate if the user is under 18 years old, it means that the value is translate from FR DateTime to OBIE LocalDate before apply any validation.

Reply:
rs-gateway-server: `com.forgerock.openbanking.aspsp.rs.api.customerinfo.v1_0.InfoApiController` line 86.
```java
if(customerInfo != null){
    LocalDate birthdate = customerInfo.getBirthdate();
    LocalDate now = LocalDate.now();
    Period age = Period.fieldDifference(birthdate, now);
    if(age.getYears() < 18){
        log.info("Customer is under 18 and is forbidden from sharing their " +
                "Customer Info. Returning '{}'", HttpStatus.FORBIDDEN);
        return ResponseEntity.status(HttpStatus.FORBIDDEN).body(null);
    }
}
```

Reply:
Ah good, thanks!